### DISCUSSION TITLE
Outlook->Google sync is using the system local time zone rather than the calendar entry time zone

### DISCUSSION THREAD

**Discussion by ajkessel** [phw198/OutlookGoogleCalendarSync#1896] [General]
I am doing one-way sync from Outlook 365 to Google. The calendar items on Google appear to be set to the time zone of the system at the time of the sync, rather than the time zone of the underlying calendar item, so if I happen to sync when I'm anywhere other than my time zone, the Google version of the event is at the wrong time. Am I doing something wrong or is this expected behavior? Is there a workaround that I'm missing?

Outlook 365 is up to date. OGCS 2.10.0.0.

**Comment by phw198** [comment]
That doesn't sound right - OGCS pulls the time zone information out of the Outlook item and syncs that to Google. Try creating your own Outlook item in a different time zone to test it. 

If the item(s) being affected are from other organisers, it may be OGCS is being blocked from querying for the organiser's time zone - later alpha versions may warn when this is happening, so it's always best to upgrade to the latest alpha when experiencing issues.

If it still doesn't seem to be working correctly for you, please reproduce the issue on a new calendar item, sync it using the latest alpha and then attach your log file to a new GitHub issue.

**Reply by ajkessel** [reply]
I'm not able to reproduce the issue now with the latest alpha, but I do think it only happens with respect to events from other organizers, not ones that I've created myself. I'll continue to test -- perhaps with the latest alpha I'll get a warning that will help me isolate the issue.

**Reply by phw198** [reply]
> it may be OGCS is being blocked from querying for the organiser's time zone

If this is the issue you are hitting, then #1962 might hold a solution once developed.
